File: LeetCode/242/solution.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        new_s = sorted(s)
        new_t = sorted(t)

        if not len(s) == len(t):
            return False

        for i in range(0, len(s)):
            if not new_s[i] == new_t[i]:
                return False
        
        return True
            

ICE = Solution()
print(ICE.isAnagram("anagram", "nagaram"))

File: LeetCode/242/solution.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isAnagram = function(s, t) {
    const len_s = s.length;
    const len_t = t.length;

    if (len_s != len_t) {
        return false;
    }

    let sorted_s = s.split("").sort().join("");
    let sorted_t = t.split("").sort().join("");

    for (let i = 0; i < len_s; i++) {
        if (sorted_s[i] != sorted_t[i]) {
            return false;
        }
    }

    return true;
};

console.log(isAnagram("anagram", "nagaram"))